src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,321 @@
+    public static final String FILE_PATH = "duke.txt";
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static final String LINE = "\t____________________________________________________________\n\t";
+    public static final String BYE = LINE
+            + "Bye. try not to procrastinate!\n"
+            + LINE;
+    public static final String GREETING = LINE
+            + "Hello! I'm Anderson\n\t"
+            + "What do you need to do?\n"
+            + LINE;
+
+    public static String getCommand(String userInput) {
+        if (userInput.length() > 2) {
+            String[] isolateCommand = userInput.split(" ");
+            return isolateCommand[0];
+        }
+        return userInput;
+    }
+
+    public static String GetItem(String userInput) throws IllegalToDoException, InvalidCommandException, EmptyCommand {
+        String item = "";
+        String command = getCommand(userInput);
+        if (isEvent(command)) {
+            item = notToDoItem(userInput);
+        } else if (isDeadline(command)) {
+            item = notToDoItem(userInput);
+        } else if (isInvalidCommand(command)) {
+            throw new InvalidCommandException();
+        } else if (userInput.length() > 3) {
+            item = getRequiredSubstring(userInput, " ", 1);
+            if (item.trim().equals("") || item.equalsIgnoreCase("todo")) {
+                throw new IllegalToDoException();
+            }
+        } else if (isEmpty(command)) {
+            throw new EmptyCommand();
+        }
+        return item;
+    }
+
+    private static String getRequiredSubstring(String userInput, String s, int i) {
+        return userInput.substring(userInput.indexOf(s) + i);
+    }
+
+    private static String notToDoItem(String userInput) {
+        return userInput.substring(userInput.indexOf(" ") + 1, userInput.indexOf("/"));
+    }
+
+    public static String getTime(String userInput) {
+        String time;
+        String command = getCommand(userInput);
+        if (isDeadline(command)) {
+            time = getRequiredSubstring(userInput, "/", 3);
+        } else if (isEvent(command)) {
+            time = getRequiredSubstring(userInput, "/", 3);
+        } else {
+            return "";
+        }
+        return time;
+    }
+
+    public static void main(String[] args) {
+
+        ArrayList<Task> unfilteredTasks = new ArrayList<Task>();
+        int unfilteredCounter = 0;
+
+        printGreetings();
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        userInput = in.nextLine();
+        boolean closeDuke;
+
+        try {
+            unfilteredTasks = loadFile(FILE_PATH, unfilteredTasks);
+        } catch (FileNotFoundException e) {
+            File dukeCheckpoint = new File(FILE_PATH);
+            try {
```

Comment:
Instead of nesting try-catch block, try to include then in the same level. 
try {
...
} catch (FileNotFoundException e) {
...
} catch (FileNotFoundException e){
...
}